lib/configure.js

```diff
@@ -383,9 +383,9 @@ function PythonFinder (configPython, callback) {
 
 PythonFinder.prototype = {
   log: logWithPrefix(log, 'find Python'),
-  argsExecutable: [ '-c', 'import sys; print(sys.executable);' ],
-  argsVersion: [ '-c', 'import sys; print("%s.%s.%s" % sys.version_info[:3]);' ],
-  semverRange: '>=2.6.0 <3.0.0',
+  argsExecutable: [ '-c', 'import sys; print(sys.executable)' ],
```

Comment:
is dropping the `;` cosmetic, or necessary? If cosmetic, I'd prefer to leave as is to not mix cosmetic and functional changes.